src/info.c

```diff
@@ -6392,6 +6404,326 @@ static const unsigned char m_butt2[]=  // Options reset button lit
    74, 74, 74, 73, 73, 73,  0,255,
 };
 
+static const unsigned char m_vbox[] =
```

Comment:
Wow, this spans over 300 lines. I'd rather have this in a separate header file. Do we have any copyright information for this? Or at least a commit when this was introduced into PrBoom?

Reply:
It's output of `M_DrawVerifyBox()` function, converted to patch format. It's mostly black, not very effective.

Reply:
But `M_DrawVerifyBox()` is a Boom/MBF function. Anyway, the other menu-related lumps are in `info.c`, do you want to refactor it?